.circleci/config.yml

```diff
@@ -875,6 +875,13 @@ jobs:
                 when: always
```

Comment:
Rename this step in both 6.0 and master to `Collect Instance Role And Check If Instance Was Created`